### PR TITLE
Partial TL translations for remote-branches.asc

### PR DESCRIPTION
**Pull request by josteinfrancois**
Here's a new pull request with 1,108 words translated. I need your review for this one. Thank you.

### CODE REVIEW

File: book/03-git-branching/sections/remote-branches.asc [hunk 1]
```diff
@@ -1,71 +1,51 @@
...
 
-Remote-tracking branches take the form `<remote>/<branch>`.
-For instance, if you wanted to see what the `master` branch on your `origin` remote looked like as of the last time you communicated with it, you would check the `origin/master` branch.
-If you were working on an issue with a partner and they pushed up an `iss53` branch, you might have your own local `iss53` branch, but the branch on the server would be represented by the remote-tracking branch `origin/iss53`.
+Ang remote-tracking na mga branch nag-aanyong `<remote>/<branch>`. Halimbawa, kung gusto mong tingnan kung ano ang mukha ng `master` na branch sa iyong `origin` na remote noong huling panahon na ikaw ay nakipag-usap nito, susuriin mo ang `origin/master` na branch. Kung ikaw ay nagtatrabaho sa isang isyu kasama ang isang kasosyo at sila ay nag-push paitaas ng isang `iss53` na branch, maaaring mayroon kang sariling lokal na `iss53` na branch, ngunit ang branch sa server ay marerepresenta ng remote-tracking na branch na `origin/iss53`.
```
Comment (severinolorillajr) [inline book/03-git-branching/sections/remote-branches.asc]: ay nag-aanyong

File: book/03-git-branching/sections/remote-branches.asc [hunk 2]
```diff
@@ -1,71 +1,51 @@
...
 
-This may be a bit confusing, so let's look at an example.
-Let's say you have a Git server on your network at `git.ourcompany.com`.
-If you clone from this, Git's `clone` command automatically names it `origin` for you, pulls down all its data, creates a pointer to where its `master` branch is, and names it `origin/master` locally.
-Git also gives you your own local `master` branch starting at the same place as origin's `master` branch, so you have something to work from.
+Maaaring ito ay medyo nakakalito, kaya tumingin tayo sa isang halimbawa. Sabihin nating mayroon kang isang Git na server sa iyong network sa `git.ourcompany.com`. Kung ikaw ay magku-clone mula nito, ang `clone` na utos ng Git ay awtomatikong papangalanan itong `origin` para sa iyo, ipu-pull nito pababa ang lahat ng data nito, gagawa ng isang pointer patungo kung saan ang `master` na branch nito, at lokal itong papangalanang `origin/master`. Ang Git ay nagbibigay din sa iyo ng iyong sariling lokal na `master` na branch sa pagsisimula sa parehong lugar bilang `master` na branch ng origin, kaya mayroon kang bagay na tatrabahuan.
```
Comment (severinolorillajr) [inline book/03-git-branching/sections/remote-branches.asc]: data to datos

File: book/03-git-branching/sections/remote-branches.asc [hunk 3]
```diff
@@ -1,71 +1,51 @@
...
 
-To demonstrate having multiple remote servers and what remote branches for those remote projects look like, let's assume you have another internal Git server that is used only for development by one of your sprint teams.
-This server is at `git.team1.ourcompany.com`.
-You can add it as a new remote reference to the project you're currently working on by running the `git remote add` command as we covered in <<_git_basics_chapter#_git_basics_chapter>>.
-Name this remote `teamone`, which will be your shortname for that whole URL.
+Upang ipakita ang pagkakaroon ng maramihang remote na mga server at kung ano ang mukha ng remote na mga branch para sa mga remote na proyekto, ating ipalagay na mayroon kang ibang panloob na Git na server na ginagamit lamang para sa pag-develop ng isa sa iyong mga sprint na mga koponan. Ang server na ito ay nasa `git.team1.ourcompany.com`. Maaari mong idagdag ito bilang isang panibagong remote na reperensya kung saan mo kasalukuyang tinatrabaho ang `git remote add` na utos na ating nasakop sa <<_git_basics_chapter#_git_basics_chapter>>. Pangalanan itong remote na `teamone`, na magiging iyong maikling pangalan para sa buong URL na iyon.
```
Comment (severinolorillajr) [inline book/03-git-branching/sections/remote-branches.asc]: mukha to hitsura

File: book/03-git-branching/sections/remote-branches.asc (hunk 1 above)
Comment (josteinfrancois) [reply book/03-git-branching/sections/remote-branches.asc]: adding `ay`

File: book/03-git-branching/sections/remote-branches.asc (hunk 2 above)
Comment (josteinfrancois) [reply book/03-git-branching/sections/remote-branches.asc]: changed data to datos

**Comment by josteinfrancois** [comment]
I have updated the pull request basing on your suggestions, @severinolorillajr. Thank you!

**Pull request merged**